widget/button.go

```diff
@@ -192,6 +194,12 @@ func (o ButtonOptions) Image(i *ButtonImage) ButtonOpt {
 	}
 }
 
+func (o ButtonOptions) IgnoreTransparentPixels(ignoreTransparentPixels bool) ButtonOpt {
```

Comment:
I am not in love with this name but cannot think of anything better. 
Please add a comment to this method explaining it.

Reply:
Added some explanation, are you happy with it?